### PR TITLE
Add write path to s3

### PR DESCRIPTION
**Pull request by smaier-ka**
Add s3 write path. Only one example file is currently written to s3. Others will follow in subsequent PRs.

### CODE REVIEW

**Comment by mretsch** [comment]
Looks all good to me. Thanks for catching the missing equal signs.
If Wwhisperer has no objections it can be merged.

File: data_scraper.py [hunk 1]
```diff
@@ -150,17 +174,26 @@ def _data_cleansing(self, df):
...
-            date_columns["Aktualisierung"]
+            date_columns=["Aktualisierung"]
         )
+
+
+def lambda_handler(message, context):
+    s3_client = boto3.client("s3")
+    hamburg_clinics = HamburgClinics()
+    response = hamburg_clinics.write_to_s3(s3_client)
+    return response
```
Comment (KnutZuidema) [inline data_scraper.py:199]: Shouldn't this write *all* datasets to s3 instead of only the HamburgClinics one??

File: data_scraper.py [hunk 2]
```diff
@@ -35,6 +46,14 @@ def _ensure_datetime(self, df):
             df[c] = pd.to_datetime(df[c], infer_datetime_format=True, format=self.date_format)
         return df
 
+    def write_to_s3(self, s3_client, bucket=BUCKET_NAME_LANDING_ZONE):
+        df = self.get_data()
+        csv_string = df.to_csv(sep=",", index=False)
+        timestamp = datetime.now().strftime("%Y-%m-%dT%H%M%S")
```
Comment (KnutZuidema) [inline data_scraper.py:52]: Could just use `datetime.isoformat()`
```suggestion
        timestamp = datetime.now().isoformat()
```

**Review by KnutZuidema — APPROVED** [review]

**Review by Wwhisperer — APPROVED** [review]
looks good to me. naming the datasets is a good change. thanks.

**Pull request merged**